ubuntudesign/vanilla-framework
# Use jekyll to build out local examples for style-guide-development

## Done

- Building on previous work by @anthonydillon (#783), I've wrapped his example partials in a minimal Jekyll scaffold.
- This removed any duplication between the example partials
- This approach provides a foundation for Percy (visual regression testing) and/or Browserstack to run a headless browser over
- This approach means styleguide-driven-development can be done entirely within this repo without depending on any other project

## QA

- Pull code
- Run `rm -rf node_modules`
- Run `npm i`
- Run `bundle install`
- You should then be able to run `gulp develop` to kick off development watcher
- Navigate to `http://127.0.0.1:4000/`
- Make a change to your local scss, save and refresh
- Profit!

Fixes #789
